app/controllers/recipe_ingredients_controller.rb

```diff
@@ -0,0 +1,7 @@
+class RecipeIngredientsController < ApplicationController
```

Comment:
Great work routing this action to a joins controller! This is the most RESTful option.